src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -344,6 +344,31 @@ public abstract partial class Attribute
         public virtual bool IsDefaultAttribute() { return default(bool); }
         public virtual bool Match(object obj) { return default(bool); }
         public virtual object TypeId { get { return default(bool); } }
+        public virtual object TypeId { get { return default(object); } }
```

Comment:
We should keep a single entry for TypeId -- and it should be the one that returns a default(object) (i.e. the one you added).
I think I added the one that mistakenly returns default(bool).

I am surprised that you did not get a build error for the duplicate property.